src/modules/dashboard/components/ActionsPage/TotalStakeWidget/GroupedTotalStake.tsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import Heading from '~core/Heading';
+import { CustomRadio } from '~core/Fields/Radio';
+import Numeral from '~core/Numeral';
+import { Form } from '~core/Fields';
+import QuestionMarkTooltip from '~core/QuestionMarkTooltip';
+
+import { getTokenDecimalsWithFallback } from '~utils/tokens';
+
+import styles from './TotalStakeWidget.css';
+
+type Props = {
+  requiredStake: string | number;
+  formattedTotalYAYStakedPercentage: string;
+  formattedTotalNAYStakedPercentage: string;
+  tokenDecimals?: number;
+  tokenSymbol?: string;
+};
+
+const MSG = defineMessages({
+  stakeProgress: {
+    id:
+      'dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.stakeProgress',
+    defaultMessage: '{totalPercentage}% of {requiredStake}',
+  },
+  crowdfundStakeTitle: {
+    id: `dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.crowdfundStakeTitle`,
+    defaultMessage: 'Crowdfund stakes',
+  },
+  YAYName: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.YAYName',
+    defaultMessage: 'Motion',
+  },
+  NAYName: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.NAYName',
+    defaultMessage: 'Objection',
+  },
+  fullyStaked: {
+    id: 'dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.fullyStaked',
+    defaultMessage: 'Fully staked',
+  },
+  totalStakeTooltip: {
+    id: `dashboard.ActionsPage.TotalStakeWidget.GroupedTotalStake.totalStakeTooltip`,
+    defaultMessage: `[TO BE ADDED WHEN AVAILABLE]`,
+  },
+});
+
+const GroupedTotalStake = ({
+  requiredStake,
+  formattedTotalYAYStakedPercentage,
+  formattedTotalNAYStakedPercentage,
+  tokenDecimals,
+  tokenSymbol,
+}: Props) => {
+  const isYAYSideFullyStaked = formattedTotalYAYStakedPercentage === '100';
+  const isNAYSideFullyStaked = formattedTotalNAYStakedPercentage === '100';
+
+  /*
+   * @NOTE
+   *
+   * A Form component is added here just to make CustomRadio happy.
+   *
+   * This isn't really a Form, and those are not really Radio buttons.
+   *
+   * The CustomRadio are just there for the visuals.
+   */
+  // eslint-disable-next-line no-console
+  const handleSubmit = () => console.log('Time to submit');
+
+  return (
+    <Form initialValues={{}} onSubmit={handleSubmit}>
+      <div className={styles.widgetHeading}>
+        <Heading
+          appearance={{
+            theme: 'dark',
+            size: 'small',
+            weight: 'bold',
+            margin: 'none',
+          }}
+          text={MSG.crowdfundStakeTitle}
+          className={styles.title}
+        />
+        <QuestionMarkTooltip
+          className={styles.tooltip}
+          tooltipText={MSG.totalStakeTooltip}
+          tooltipPopperProps={{
+            placement: 'right',
+          }}
+        />
+      </div>
+      <div className={styles.totalStakeRadio}>
+        <CustomRadio
```

Comment:
There needs to be more left padding for the stake radio buttons labels:

Yours:
![Screenshot from 2021-04-26 10-53-42](https://user-images.githubusercontent.com/1193222/116049311-d2f08400-a67e-11eb-9119-13f9d8b36372.png)

Spec:
![Screenshot from 2021-04-26 10-53-48](https://user-images.githubusercontent.com/1193222/116049313-d2f08400-a67e-11eb-8800-37b652ed5a0d.png)